content/file/file.go

```diff
@@ -349,6 +349,29 @@ func (s *Store) Resolve(ctx context.Context, ref string) (ocispec.Descriptor, er
 	return s.resolver.Resolve(ctx, ref)
 }
 
+func (s *Store) Delete(ctx context.Context, target ocispec.Descriptor) error {
+	if s.isClosedSet() {
+		return ErrStoreClosed
+	}
+	exists, err := s.Exists(ctx, target)
+	if err != nil {
+		return err
+	}
+	if !exists {
+		return errdef.ErrNotFound
+	}
+	path, _ := s.digestToPath.Load(target.Digest)
+	pathString, ok := path.(string)
+	if !ok {
+		return errdef.ErrInvalidDigest
+	}
+	err = os.RemoveAll(s.absPath(pathString))
```

Comment:
If `digest` cannot be found in `digestToPath`, we should head to the `fallbackStorage` directly.